src/coreclr/tools/Common/Compiler/DependencyAnalysis/ObjectNodeSection.cs

```diff
@@ -47,7 +47,7 @@ public bool IsStandardSection
         public static readonly ObjectNodeSection FoldableReadOnlyDataSection = new ObjectNodeSection("rdata", SectionType.ReadOnly);
         public static readonly ObjectNodeSection TextSection = new ObjectNodeSection("text", SectionType.Executable);
         public static readonly ObjectNodeSection TLSSection = new ObjectNodeSection("TLS", SectionType.Writeable);
-        public static readonly ObjectNodeSection BssSection = new ObjectNodeSection("bss", SectionType.Writeable);
+        public static readonly ObjectNodeSection BssSection = new ObjectNodeSection("bss", SectionType.Uninitialized);
```

Comment:
I've confirmed in the CI logs this also fixes the linker warning in multimodule testing: `MultiModule.obj : warning LNK4078: multiple '.bss' sections found with different attributes (C0401040)`. Yay.